### PR TITLE
bshultz-toast/DOC-1935

### PR DESCRIPTION
**Pull request by bshultz-toast**
Fixed a small typo in "restautant", updated the path and condensed the description

### CODE REVIEW

**Comment by bshultz-toast** [comment]
Updated to reflect new thoughts emerging on reusing HTTP method names

File: openapi/toast-partners-api.yaml [hunk 1]
```diff
@@ -203,7 +203,8 @@ definitions:
 paths:
   /restaurants:
     get:
-      summary: Returns information about restaurants a partner API client has access to.
+      operationId: restaurantsGet
+      summary: Get accessible restaurants' details.
```
Comment (pdesjardins) [inline openapi/toast-partners-api.yaml:207]: Would it be simpler to use `Get accessible restaurant information`?

**Review by pdesjardins — APPROVED** [review]
The operationId looks great. I left a comment about a possible further simplification of the summary string. This looks good to merge to me. Thanks! 👍

File: openapi/toast-partners-api.yaml (hunk 1 above)
Comment (bshultz-toast) [reply openapi/toast-partners-api.yaml:207]: Well yeah, trying to stay within the character count limitation had me getting creative with the diction before we changed course and said it was cool to repeat the HTTP method...

**Comment by bshultz-toast** [comment]
updated summary

**Pull request merged**